dev-tools/scripts/scriptutil.py

```diff
@@ -99,6 +108,18 @@ def update_file(filename, line_re, edit):
     f.write(''.join(buffer))
   return True
 
+
+def check_ant():
```

Comment:
Moved from buildAndPushRelease. Now also returns the actual ANT version. Will be used in releaseWizard script for warning about certain Ant versions.